ros/noetic.v3.17/ros-noetic-behaviortree-cpp-v3/apkbuild_hook.sh

```diff
@@ -0,0 +1,3 @@
+apkbuild_hook() {
+  makedepends="${makedepends} ros-noetic-catkin>=0.5.78 ros-noetic-roslib"
```

Comment:
https://github.com/BehaviorTree/BehaviorTree.CPP/blob/04a514eecc0e02e38f8c104084f43a85f57fabd2/package.xml#L18-L19
```
  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
  <depend condition="$ROS_VERSION == 1">roslib</depend>
```
This seems be discarded.
I guess setting `ROS_VERSION` in https://github.com/seqsense/aports-ros-updater/blob/master/Dockerfile will fix this.